ui-testing-core/src/main/java/com/avito/android/test/fps/FpsMeasureFrameCallback.kt

```diff
@@ -0,0 +1,53 @@
+package com.avito.android.test.fps
+
+import android.content.Context
+import android.view.Choreographer
+import android.view.WindowManager
+import java.util.concurrent.TimeUnit
+
+class FpsMeasureFrameCallback(
+    context: Context,
+    measureFactor: Int,
+    private val choreographer: Choreographer,
+    private val report: (Long) -> Unit
+) : Choreographer.FrameCallback {
```

Comment:
I don't see any reason to add this class to the library. What is its purpose?

Reply:
To Have default class for fps measuring purposes. Where it should be placed instead?